client/react/component/ClientOverviewTab.js

```diff
@@ -0,0 +1,102 @@
+import moment from 'moment';
+import React from 'react';
+import { createContainer } from 'meteor/react-meteor-data';
+import { Clients } from '../../../imports/api/clients/clients';
+// import { Questions } from '../../../imports/api/questions/questions';
+import { getRace, getGender, getEthnicity, getYesNo } from
+'../../../imports/ui/clients/textHelpers.js';
+
+class ClientOverviewTab extends React.Component {
+  constructor() {
+    super();
+    this.state = {};
+    this.edit = this.edit.bind(this);
+  }
+
+  getText(text, code) {
```

Comment:
might be a better option to move this `getText` function to `textHelpers.js` instead of other function imports